src/gui/editors/PianoRoll.cpp

```diff
@@ -2439,7 +2539,7 @@ void PianoRoll::mouseMoveEvent( QMouseEvent * me )
 				}
 			}
 		}
-		else if (me->buttons() == Qt::NoButton && m_editMode != ModeDraw)
+		else if (me->buttons() == Qt::NoButton && m_editMode != ModeDraw && m_editMode != ModeEditRazor)
```

Comment:
I think this is not required, unless there is something I am overlooking. (if so, please do enlighten me)

Reply:
Try moving the mouse horizontal over a note while in razor mode; you'll see that the cursor starts flickering without that check.

Reply:
Ah, I hadn't noticed the bug. Thanks for informing me!